src/components/ContentOpenWith/__tests__/ExecuteForm-test.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import ExecuteForm from '../ExecuteForm';
+
+describe('components/ContentOpenWith/ExecuteForm', () => {
+    const executePostData = {
+        url: 'foo.com',
+        params: [{ key: 'foo', value: 'bar' }, { key: 'bar', value: 'baz' }],
+    };
+
+    const submitStub = jest.fn();
+    const ref = {
+        current: {
+            submit: submitStub,
+        },
+    };
+    const onSubmitStub = jest.fn();
+
+    const getWrapper = props => shallow(<ExecuteForm {...props} />);
```

Comment:
It might be simpler to use `mount` since you're dealing with refs, rather than unwrapping the instance and setting the property manually.